pkg/cmd/secret.go

```diff
@@ -0,0 +1,342 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/puppetlabs/relay/pkg/debug"
+	"github.com/puppetlabs/relay/pkg/errors"
+	"github.com/puppetlabs/relay/pkg/format"
+	"github.com/puppetlabs/relay/pkg/model"
+	"github.com/puppetlabs/relay/pkg/util"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func newSecretCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "secret",
+		Short: "Manage your Relay secrets",
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	// TODO Do we want just set, or add/update?
```

Comment:
I'll just delete the add/update or set functions before merge, but wanted to double-check that using `set` was the right move.